src/components/StatusBadge/StatusBadge.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import StatusBadge from './StatusBadge';
+
+export default {
+  title: 'StatusBadge',
+  component: StatusBadge,
+  argTypes: {
+    status: { control: 'inline-radio', options: ['UNPAID', 'PARTIAL', 'PAID'] },
```

Comment:
In the Figma seems like the Status Badge component is always in capital letters. Can we change the component to uppercase the `status` param and pass a normal capitalised word to see if works correctly?